packages/svelteui-core/src/lib/components/NumberInput/NumberInput.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import { createEventDispatcher } from 'svelte';
 	import { css, dark } from '$lib/styles';
-	import Input from '../Input/Input.svelte';
+	import TextInput from '../TextInput/TextInput.svelte';
```

Comment:
- when importing components use named imports. not default ones

example:
` import { TextInput } from '../TextInput' `